group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupCoordinatorConfigTest.java

```diff
@@ -44,6 +44,19 @@ public class GroupCoordinatorConfigTest {
             GroupCoordinatorConfig.CONSUMER_GROUP_CONFIG_DEF,
             GroupCoordinatorConfig.SHARE_GROUP_CONFIG_DEF);
 
+    @Test
+    public void testConsumerGroupAssignorsDefault() {
+        // The full class name of the assignors if part of our public api. Hence,
```

Comment:
```suggestion
        // The full class name of the assignors is part of our public api. Hence,
```
There's a small typo here